__tests__/app.test.js

```diff
@@ -141,3 +141,80 @@ describe("GET /api/articles", () => {
     })
 })
 
+describe("POST /api/articles/:article_id/comments", () => {
+    test('inserts a new comment to the db and sends back the new comment', () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: "this is a test"
+        }
+        return request(app)
+        .post('/api/articles/3/comments')
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+            const newComm = response.body.comment
+            expect(newComm.author).toBe('butter_bridge');
+            expect(newComm.body).toBe('this is a test');
+            expect(newComm.article_id).toBe(3);
+            expect(newComm.votes).toBe(0);
```

Comment:
This is a good test, you could also add an assertion that you expect the property `created_at` to exist on the object